Lib/test/test_types.py

```diff
@@ -2345,5 +2346,40 @@ def ex(a, /, b, *, c):
             )
 
 
+class SubinterpreterTests(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        global interpreters
+        try:
+            from test.support import interpreters
+        except ModuleNotFoundError:
+            raise unittest.Skip('subinterpreters required')
+        import test.support.interpreters.channels
+
+    @cpython_only
+    def test_slot_wrappers(self):
+        rch, sch = interpreters.channels.create()
+
+        # For now it's sufficient to check int.__str__.
+        # See https://github.com/python/cpython/issues/117482
+        # and https://github.com/python/cpython/pull/117660.
```

Comment:
The link should be https://github.com/python/cpython/pull/121602 in this case

Reply:
The link is for the great analysis you did in gh-117660, rather than about any fix.

Reply:
Ok, thank you!!